lmnet/README.md

```diff
@@ -41,9 +41,11 @@ cd dataset/CIFAR_10 && tar xvf cifar-10-python.tar.gz && cd ../..
 python -m venv .venv
 source .venv/bin/activate
 
+# We need to install cython and numpy before "pip install -r" for pycocotools
+pip install --upgrade cython numpy
+
 # install python requirements
 pip install -r ../cpu.requirements.txt
-pip install -e third_party/coco/PythonAPI
```

Comment:
Isn't it enough to change this line to `pip install pycocotools==2.0.0` instead of adding `pip install --upgrade cython numpy`?

Reply:
@hadusam 
`pip install pycocotools==2.0.0` failed with 

```
 pip install pycocotools==2.0.0
Collecting pycocotools==2.0.0
  Using cached https://files.pythonhosted.org/packages/96/84/9a07b1095fd8555ba3f3d519517c8743c2554a245f9476e5e39869f948d2/pycocotools-2.0.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-dka62yx8/pycocotools/setup.py", line 2, in <module>
        from Cython.Build import cythonize
    ModuleNotFoundError: No module named 'Cython'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-dka62yx8/pycocotools/
You are using pip version 18.1, however version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Reply:
Thanks! I think adding `Cython` into `requirements.txt` will solve the above error, won't it?

Reply:
Oh sorry, there is a comment, I understood. https://github.com/blue-oil/blueoil/pull/775#issuecomment-580058375

Reply:
But I don't understand why... 🤔 I will check it.

Reply:
NO

Reply:
I write just two line requirements.txt

```
cython==0.27.3
pycocotools==2.0.0
```

and `pip install -r requirements.txt`. It's failed with

```$ pip install -r requirementst.txt
Collecting cython==0.27.3 (from -r requirementst.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/e9/91/46cb3f4c73f1e96faa517f96e9d12de5b8c97d404c7ab71553da0e58c980/Cython-0.27.3-cp36-cp36m-manylinux1_x86_64.whl
Collecting pycocotools==2.0.0 (from -r requirementst.txt (line 2))
  Using cached https://files.pythonhosted.org/packages/96/84/9a07b1095fd8555ba3f3d519517c8743c2554a245f9476e5e39869f948d2/pycocotools-2.0.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-7h2dg1j0/pycocotools/setup.py", line 2, in <module>
        from Cython.Build import cythonize
    ModuleNotFoundError: No module named 'Cython'

```

Reply:
Thanks. @iizukak 
In my opinion, removing `pycocotools` from `requirements.txt` is better because `pycocotools` is additional dependency only for `coco` dataset.
Especially it is not good for me that manually dependency installation of `numpy` put outside of `requirements.txt`. If manually installation is inevitable, I think it is better to choose `pycocotools` instead of `numpy`, `cython`.

So. how about adding a line into requirements.txt?
```
cython==0.27.3
```
and `pip install -r requirements.txt`, after that, `pip install pycocotools==2.0.0` manually.
🤔

Reply:
@hadusam Agreed.

Reply:
@hadusam Changed. Please check.